GSA/digitalgov.gov
# Update youtube shortcode to accept timestamp

## Summary
Current `youtube` shortcode offers no option to set the start time for an embedded youtube video.

Why? Some videos from the [accessibility-for-teams](https://github.com/GSA/digitalgov.gov/pull/7279) guide migration use specific start times to accompany text content.
It would be useful to keep these video bookmarks after the migration and it will also provide the functionality in the future if needed.

### Preview

[Link to Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/nl-refactor-youtube-shortcode/event/2018/09/27/a-conversation-with-evan-papp/)

### Solution

This enhancement adds two fields for setting the required `id` and optional `time`.

**Before**

```
{{< youtube Hvo7WVKvPGU >}}
```

**After**
```
{{< youtube id="Hvo7WVKvPGU" title="Youtube video title" start_time="101" >}}
```

The `title` field sets the html `title` attribute on the `iframe` to meet accessibility.


### Checklist

- [x] updated shortcode partial
- [x] updated existing shortcode instances
- [x] documented enhancements in code
- [ ] updated wiki


### How To Test

1. Visit [page with youtube embed](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/nl-refactor-youtube-shortcode/event/2018/09/27/a-conversation-with-evan-papp/) (the second video on the page)
2. Add `start_time` to shortcode on local build
3. Visit page, press play and video should start at provided time

---

### Dev Checklist
